setup.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# This file is part account_es module for Tryton.
+# The COPYRIGHT file at the top level of this repository contains
+# the full copyright notices and license terms.
+
+from setuptools import setup
+import re
+import os
+import io
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+
+MODULE = 'create_transactions'
+PREFIX = 'trytoncalidae'
+MODULE2PREFIX = {
+    'account_bank_statement': 'trytonspain',
+    'account_bank_statement_account': 'trytonspain',
+    'analytic_bank_statement': 'nantic',
+}
+
+
+def read(fname):
+    return io.open(
+        os.path.join(os.path.dirname(__file__), fname),
+        'r', encoding='utf-8').read()
+
+
+def get_require_version(name):
+    if minor_version % 2:
+        require = '%s >= %s.%s.dev0, < %s.%s'
+    else:
+        require = '%s >= %s.%s, < %s.%s'
+    require %= (
+        name, major_version, minor_version,
+        major_version, minor_version + 1
+    )
+    return require
+
+
+config = ConfigParser()
+config.readfp(open('tryton.cfg'))
+info = dict(config.items('tryton'))
+for key in ('depends', 'extras_depend', 'xml'):
+    if key in info:
+        info[key] = info[key].strip().splitlines()
+version = info.get('version', '0.0.1')
+major_version, minor_version, _ = version.split('.', 2)
+major_version = int(major_version)
+minor_version = int(minor_version)
+
+requires = [
```

Comment:
This is intended for python requirements that are imported in our sources. Requests, jinja2, etc. The place where to find those dependencies should not be specified in the setup.py file, it is responsibility for whoever installs this software to provide a package index or find-links option where trytonspain-account_bank_statement exists!